bindings/ruby/lib/rock_gazebo/syskit/configuration_extension.rb

```diff
@@ -98,10 +98,16 @@ def use_gazebo_world(
                             {}
                         end
 
-                    ::Syskit.conf.register_process_server(
-                        "gazebo", ::Syskit::RobyApp::UnmanagedTasksManager.new,
-                        app.log_dir, **options
-                    )
+                    if ::Syskit.conf.respond_to?(:register_unmanaged_manager)
```

Comment:
```suggestion
                    if ::Syskit.conf.respond_to?(:create_unmanaged_manager)
```

Reply:
Was the opposite ... the right one is `register_`